cmd/root.go

```diff
@@ -20,14 +19,14 @@ var (
 	Version = "development-build"
 )
 
-
 // NewRootCmd Create new root command
 func NewRootCmd() *cobra.Command {
 	cmd := cobra.Command{
 		Use:   "goft",
 		Short: "CLI tool to interact with 42's API",
 	}
-	cmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.goft.yaml)")
+	defaultconf := os.Getenv("HOME") + "/.config/goft/config.yml"
+	cmd.PersistentFlags().StringVar(&cfgFile, "config", defaultconf, "config file")
```

Comment:
The flag's description should still say where the config path is located

Reply:
`cobra` makes description with default value. We don't need to write default value in the description sentence.
```
$ goft -h
CLI tool to interact with 42's API

Usage:
  goft [command]

Available Commands:
  agu         Manage Anti Grav Units (AGUs)
  closes      Manage users' closes
  help        Help about any command
  requests    Send raw requests to the Intranet's API
  users       Interact with the user entity

Flags:
      --config string   config file (default "/Users/tkomatsu/.config/goft/config.yml")
  -h, --help            help for goft
  -v, --version         version for goft

Use "goft [command] --help" for more information about a command.
```